integration/tests/tracer/test_tracer_debug_trace_transactions_callTracer.py

```diff
@@ -0,0 +1,501 @@
+import random
+
+import allure
+import pytest
+import json
+
+from deepdiff import DeepDiff
+from utils.helpers import wait_condition
+from integration.tests.basic.helpers.basic import AccountData
+from utils.web3client import NeonChainWeb3Client
+from utils.accounts import EthAccounts
+from utils.apiclient import JsonRPCSession
+from integration.tests.tracer.test_tracer_historical_methods import call_storage
+
+@allure.feature("Tracer API")
+@allure.story("Tracer API RPC calls debug method trace_transaction callTracer check")
+@pytest.mark.usefixtures("accounts", "web3_client", "tracer_api")
+class TestDebugTraceTransactionCallTracer:
+    web3_client: NeonChainWeb3Client
+    accounts: EthAccounts
+    tracer_api: JsonRPCSession
+
+
+    def fill_expected_response(self, instruction_tx, receipt, 
+                               logs=False, 
+                               revert=False, 
+                               revert_reason=None, 
+                               calls_value="0x1", 
+                               calls_type="CALL", 
+                               calls_logs_append=False
+                               ):
+        expected_response = {}
+        expected_response["calls"] = []
+
+        address_to = instruction_tx["to"].lower()
+        expected_response["from"] = instruction_tx["from"].lower()
+        expected_response["to"] = address_to
+        expected_response["gasUsed"] =  hex(receipt["gasUsed"])
+        expected_response["input"] = instruction_tx["data"]
+        expected_response["type"] = "CALL"
+
+        if logs:
+            for log in receipt["logs"]:
+                if log["logIndex"] == 0:
```

Comment:
So, you are checking only 2 log events (1 external and 1 inner) 
Should we have a test to check case when there are more than 1 event ?

Reply:
another one case is added: test_callTracer_call_contract_with_event_from_other_one_with_two_events